cryptconverter/plugin/main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"github.com/hashicorp/go-plugin"
+	"github.com/temporalio/samples-go/cryptconverter"
+	"go.temporal.io/sdk/converter"
+	"go.temporal.io/server/tools/cli"
+)
+
+var handshakeConfig = plugin.HandshakeConfig{
+	ProtocolVersion:  1,
+	MagicCookieKey:   "TEMPORAL_PLUGIN_DATA_CONVERTER",
+	MagicCookieValue: "cookie",
```

Comment:
I think it's probably best practice to make this a little more application-specific, even though this is a sample people are probably going to copy-paste things like this wholesale